staging/src/k8s.io/client-go/examples/leader-election/main.go

```diff
@@ -124,38 +121,29 @@ func main() {
 		// get elected before your background loop finished, violating
 		// the stated goal of the lease.
 		ReleaseOnCancel: true,
-		LeaseDuration:   60 * time.Second,
-		RenewDeadline:   15 * time.Second,
-		RetryPeriod:     5 * time.Second,
+		LeaseDuration:   15 * time.Second,
+		RenewDeadline:   10 * time.Second,
+		RetryPeriod:     2 * time.Second,
```

Comment:
what is the reason for these changes?

Reply:
From kube-controller-manager, and we can adjust the value according to personal consideration.

Reply:
as an example, a one minute lease seems ok